oneanddone/base/templatetags/jinja_helpers.py

```diff
@@ -0,0 +1,116 @@
+import datetime
```

Comment:
Thanks for taking a look at this commit, @robhudson. I appreciate this. I will look into this alternate solution and open a bug to update the code. Also, @glogiotatidis asked me to submit a PR to https://github.com/mozilla/sugardough to add this `fe` filter, which I did. I will update that PR to mention this alternate approach.